client/package.json

```diff
@@ -32,8 +33,8 @@
     "diagram-js": "^7.8.2",
     "diagram-js-direct-editing": "^1.6.3",
     "diagram-js-origin": "^1.3.2",
-    "dmn-js": "^11.1.2",
-    "dmn-js-properties-panel": "^0.7.0",
+    "dmn-js": "^12.1.0",
```

Comment:
Just basic understanding question:
Why do we still depend on dmn-js and dmn-js-properties-panel, when we also import camunda-dmn-js?

Reply:
Probably because the overview feature: https://github.com/camunda/camunda-modeler/tree/develop/client/src/app/tabs/dmn/modeler/features/overview

Reply:
I wonder whether this could cause trouble in the future when the DMN Tab is using the dmn-js version of camunda-dmn-js, but the overview is using another one.

Reply:
Or are those peer-dependencies for camunda-dmn-js?

Reply:
It's this way because we depend on the dmn-js subpackages in multiple places in the app. But I will look into this again, perhaps `dmn-js` should be a peer dependency of `camunda-dmn-js`.

Reply:
Regarding dmn-js-properties-panel, we will need to depend on it for correct plugin support. We will need to expose the properties panel so that `useService` works.

Reply:
Cf. how this works for the bpmn-js-properties-panel.

Reply:
Yeah right, for camunda-bpmn-js we have the same challenge. 

> But I will look into this again, perhaps dmn-js should be a peer dependency of camunda-dmn-js.

Makes sense to me

Reply:
https://github.com/camunda/camunda-dmn-js/pull/16

Reply:
Hmm perhaps the build-on-demand will work nicely with dmn-js when we peer-depend on it in camunda-dmn-js ^^